functions/global-setup.ts

```diff
@@ -6,4 +6,14 @@ module.exports = async function globalSetup() {
     port: 3000,
     launchTimeout: 50000,
   })
+  // Waiting for wrangler to return a request before running tests
+  for (let i = 0; i < 3; i++) {
+    const res = await fetch(new Request('http://127.0.0.1:3000/tokens/ethereum/NATIVE'))
+    if (res.ok) {
+      return
+    }
+    //Set timeout to make sure the server isn't flooded with requests if wrangler is not running
```

Comment:
```suggestion
    // Set timeout to make sure the server isn't flooded with requests if wrangler is not running
```

These files don't look like they're being linted properly. Could you also configure .eslintrc to make sure that these files are included?